core-services/prow/02_config/rh-ecosystem-edge/kernel-module-management/_prowconfig.yaml

```diff
@@ -0,0 +1,15 @@
+tide:
+  queries:
+  - labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
```

Comment:
Think we can drop this one as we are moving to Jira no?!

Reply:
I think so. I was looking for the docs of individual plugins before deleting anything, but counldn't find it.

Reply:
I guess that we can always merge that way. If we don't use BZs it will be ignored anyway. (I mean if that file was generated by some command).

Reply:
Let us merge and remove items as necessary in the future.